### ISSUE TITLE
Bump liberty adapter base image to support Java 11 or 17

### ISSUE THREAD

**Issue by rafaeltuelho** [MicroShed/microshed-testing#291]
**Describe the bug**
Current openliberty base image used by liberty adapter uses OpenJ9 1.8

**To Reproduce**
Use microprofile 6.0 feature in your app

**Expected behavior**

**System information:**
 - OS: Mac
 - Java Version: 11
 - MicroShed Testing Version: 0.9.1

**Additional context**
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running dev.odo.starter.StarterApplicationIT
[INFO] Found application file at: /Users/rsoares/dev/github/rafaeltuelho/openliberty-devfile-stack-starters/target/openliberty-starter-app.war
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[INFO] Using ServerAdapter: org.testcontainers.containers.liberty.LibertyAdapter
[INFO] Using ApplicationEnvironment class: org.microshed.testing.testcontainers.config.TestcontainersConfiguration
[INFO] Starting 1 container(s) in parallel for class dev.odo.starter.StarterApplicationIT
SLF4J: Failed to load class "org.slf4j.impl.StaticMDCBinder".
SLF4J: Defaulting to no-operation MDCAdapter implementation.
SLF4J: See http://www.slf4j.org/codes.html#no_static_mdc_binder for further details.
[INFO]   localhost/testcontainers/mtjfucdtpnpywc3b:latest
[INFO] ApplicationContainer[localhost/testcontainers/mtjfucdtpnpywc3b:latest] has exposed ports:
[INFO]   9080 --> 49164
[ApplicationContainer] 
[ApplicationContainer] Launching defaultServer (Open Liberty 23.0.0.5/wlp-1.0.77.cl230520230514-1901) on Eclipse OpenJ9 VM, version 1.8.0_372-b07 (en_US)
[ApplicationContainer] [AUDIT   ] CWWKE0001I: The server defaultServer has been launched.
[ApplicationContainer] [AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/keystore.xml
[ApplicationContainer] [AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/open-default-port.xml
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.servlet.api-6.0 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.jakarta.expressionLanguage-5.0 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.jsonpImpl-2.1.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The com.ibm.websphere.appserver.eeCompatible-10.0 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The restfulWSClient-3.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.activation.internal-2.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.jakarta.persistence-3.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.jakarta.persistence.base-3.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.jakarta.interceptor-2.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The cdi-4.0 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.jakarta.restfulWS-3.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The mpMetrics-5.0 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.jakarta.mail-2.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.org.eclipse.microprofile.openapi-3.1 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.restfulWS3.1-mpMetrics-monitor1.0 feature requires a minimum Java runtime environment version of JavaSE 11.
[ApplicationContainer] [ERROR   ] CWWKF0032E: The io.openliberty.jakarta.cdi-4.0 feature requires a minimum Java runtime environment version of JavaSE 11.
```

**Comment by KyleAure** [comment]
https://github.com/MicroShed/microshed-testing/blob/d042216589f84a60591787c11e2a04bf11006369/modules/liberty/src/main/java/org/testcontainers/containers/liberty/LibertyAdapter.java#LL46C1-L46C96

This is the base image.  It is meant to be overridden if you want to use a higher level of java or newer features. 
Though, you may still run into this issue: https://github.com/MicroShed/microshed-testing/issues/292

**Comment by rafaeltuelho** [comment]
Yeah, I figured out how to override the base image, but I hit some dependencies issues (classloader hell) at testing runtime. Microshed uses CXF libraries for REST... Anyways, I gave up for a moment.

**Comment by appiepollo14** [comment]
@rafaeltuelho is merged into main, hopefully in a futurerelease

**Comment by appiepollo14** [comment]
@KyleAure can be closed, is implemented in main.
